src/gui/screen_lan_settings.c

```diff
@@ -125,18 +127,17 @@ static void screen_lan_settings_init(screen_t *screen) {
     //============== DECLARE VARIABLES ================
 
     plan_str = (char *)gui_malloc(150 * sizeof(char));
-    char mac_addr_str[18];
 
     //============= FILL VARIABLES ============
 
-    _parse_MAC_addr(mac_addr_str);
+    _parse_MAC_addr(plsd->mac_addr_str);
```

Comment:
why "_function_name" and "function_name"? why some functions starts with under score?

Reply:
These are just "private" functions, that are not included in header and are used only in this source file.